src/systems/filecoin_blockchain/storage_power_consensus/storage_power_consensus_subsystem.go

```diff
@@ -73,17 +73,17 @@ func (spc *StoragePowerConsensusSubsystem_I) StoragePowerConsensusError(errMsg s
 	panic("TODO")
 }
 
-func (spc *StoragePowerConsensusSubsystem_I) IsWinningChallengeTicket(challengeTicket sector.ChallengeTicket) bool {
+func (spc *StoragePowerConsensusSubsystem_I) IsWinningPartialTicket(partialTicket sector.PartialTicket) bool {
 
 	// TODO: Send message to SPA to get ActivePowerInSector associated with sector
 	activePowerInSector := uint64(block.StoragePower(0))
 
 	// TODO: Send message to SPA
 	totalPower := uint64(block.StoragePower(0))
 
-	electionProof := block.SHA256(challengeTicket.Ticket())
+	challengeTicket := block.SHA256(partialTicket)
 
-	return spc.ec().IsWinningElectionProof(electionProof, activePowerInSector, totalPower)
+	return spc.ec().IsWinningChallengeTicket(challengeTicket, activePowerInSector, totalPower)
```

Comment:
Given the distinction between Active and Inactive power and inactive power (committed, recovering, and failing) requires pledge collateral, we might want to do a write up so that we are fully aligned and understood the protocol impact here. I can be the one writing this up too, cc @jzimmerman .

Reply:
Happy to review if you do @zixuanzh. I have less of a sense than you here.